### ISSUE TITLE
Android Basics: Introduction to Coroutines in Kotlin Playground

### ISSUE THREAD

**Issue by hhasert** [google-developer-training/basic-android-kotlin-compose-training-race-tracker#28]
Actually Kotlin playground executes with the delay, but just prints everything in the box when it is finished, so I cannot see the effect of the delay properly.

**Comment by scharffc** [comment]
Yes, I noticed too that it is not possible to see the delay when executing in the playground.

**Comment by purezen** [comment]
Same here. Have you found a way to test it ?

**Comment by ZioYuri78** [comment]
Did they test what they do?
I don't think so, all the learning paths i followed until now are full of problems like this.

**Comment by purezen** [comment]
It just looks really outdated. Don't think these are maintained.
